functests/performance/updating_profile.go

```diff
@@ -133,6 +138,26 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 				}
 			}
 		})
+
+		It("Reverts back all profile configuration", func() {
+			// BUG: CNF-385. Workaround - we have to remove hugepages first
+			profile.Spec.HugePages = nil
+			Expect(testclient.Client.Update(context.TODO(), profile)).ToNot(HaveOccurred())
```

Comment:
Can you try to fetch the updated profile one via `Get` before the update, I hope it will solve the problem.

Reply:
using `Patch` is eben better, see #179 as example

Reply:
hh I unsure if it more readable

Reply:
I don't need to get new profile **here** because I already got it from BeforeEach()

But later (starting from line 149) I have 2 options: either I fetch profile and use Update() or I'm just using Patch()

With Update() we will have less code but one additional request to the cluster.

Reply:
> I don't need to get new profile **here** because I already got it from BeforeEach()

ah yes :+1: 

> 
> But later (starting from line 149) I have 2 options: either I fetch profile and use Update() or I'm just using Patch()
> 
> I'm not really sure which way is better.
> With Update() we will have less code but one additional request to the cluster.

`Patch()` :+1: 
As discussed in the context of the other PR: while in a test it's not that relevant to reduce API calls, I consider using `Patch()` best practice, and we should use it everywhere now where needed.

/lgtm
/approve